apps/dashboard/src/routes/courses/[id]/lessons/+page.svelte

```diff
@@ -11,14 +11,16 @@
   import { handleDelete, lessons, lessonSections } from '$lib/components/Course/components/Lesson/store/lessons';
   import { course } from '$lib/components/Course/store';
   import { CourseContainer } from '$lib/components/CourseContainer';
-  import { PageBody, PageNav } from '$lib/components/Page';
+  import { PageBody } from '$lib/components/Page';
```

Comment:
We shouldn't be using PageBody from here but inside the structure of the Page component.